sixpack.js

```diff
@@ -197,7 +198,7 @@
             document.body.appendChild(script);
         } else {
             var http = require('http');
-            var req = http.get(url, function(res) {
+            var req = http.get(url, { headers: { 'Cookie': cookie } }, function(res) {
```

Comment:
Can you add a test for this?

Reply:
I certainly can... I did avoid it on purpose after trying a bit because the current setup had little support for mocks and such. But you're right, we should do it. 😓  I'll take a look.

Reply:
[Test added](https://github.com/jusbrasil/sixpack-js/pull/3/commits/5ed4da98b26db886a7a6966dff01ac84dea37095#diff-9f1eadd86e5db3c722b425a6f6ab5611R31), thank you for the feedback! 8)